howso/utilities/tests/test_features.py

```diff
@@ -215,13 +217,14 @@ def test_date_feature_serialization(
     valid_dtype,
     expected_data
 ):
+    """Test the serialization of date features."""
```

Comment:
Could we refactor this test to have "default_tz" as another parametrize parameter and, when it is unset, check to make sure the warnings _are_ raised?